sanowret-crystal.lic

```diff
@@ -0,0 +1,39 @@
+=begin
+	Runs in the background, gazing or exhaling your sanowret crystal. Checks for concentration and current mindstate.
+
+	author: NBSL
+	game: dr
+  version: 0.1 (2017-06-12)
+
+  changelog:
+    0.1 (2017-06-12)
+     * Initial Release
+=end
+
+no_kill_all
+no_pause_all
+
+custom_require.call(%w[common drinfomon])
+invalid_room = nil
+
+while true do
+  line = script.gets?
+  response = ""
+  if DRStats.concentration == 100 && !hidden? && invalid_room != Room.current.id
+    if DRSkill.getxp('Arcana') <= 10
+      reponse = DRC.bput("gaze sanowret crystal", "A soft light blossoms in the very center of the crystal, and begins to fill your mind with the knowledge of \w+.", "However, nothing much else happens, as you lack the concentration to focus.", "This is not a good place for that.")
```

Comment:
There's not likely to be any support here for non worn crystals, the whole point of wearing them is they work during RT and basically no matter what you're doing, not wearing it ruins that.